### ISSUE TITLE
Error While Loading nsynth Dataset

### ISSUE THREAD

**Issue by captain-pool** [tensorflow/datasets#170]
**Short description**
Error Loading nsynth Dataset

**Environment information**
* Operating System: Debian 4.9.130-2
* Python version: Python 3.5
* `tensorflow-datasets`/`tfds-nightly` version: tensorflow-datasets==1.0.1
* `tensorflow`/`tensorflow-gpu`/`tf-nightly`/`tf-nightly-gpu` version: 1.13.1

**Reproduction instructions**

```
import tensorflow_datasets as tfds
ds = tfds.load(name="nsynth", split=["train"])
```

**Logs**
```File "/home/rick/tfds/env/lib/python3.5/site-packages/tensorflow_datasets-1.0.1-py3.5.egg/tensorflow_datasets/core/dataset_builder.py", line 645, in generator_fn
    yield self.info.features.encode_example(ex)
  File "/home/rick/tfds/env/lib/python3.5/site-packages/tensorflow_datasets-1.0.1-py3.5.egg/tensorflow_datasets/core/features/feature.py", line 478, in encode_example
    encoded_feature = feature.encode_example(example_value)
  File "/home/rick/tfds/env/lib/python3.5/site-packages/tensorflow_datasets-1.0.1-py3.5.egg/tensorflow_datasets/core/features/feature.py", line 478, in encode_example
    encoded_feature = feature.encode_example(example_value)
  File "/home/rick/tfds/env/lib/python3.5/site-packages/tensorflow_datasets-1.0.1-py3.5.egg/tensorflow_datasets/core/features/class_label_feature.py", line 143, in encode_example
    if not -1 <= example_data < self._num_classes:
TypeError: unorderable types: int() <= bytes()
```

**Expected behavior**
No Errors Expected

**Comment by captain-pool** [comment]
@cyfra  @rsepassi  @dynamicwebpaige please take a look. I'm open to work on this, so can you assign it to me?

**Comment by cyfra** [comment]
This seems to be the issue with python3. (as python 2.7 seems to work fine).

Thanks for taking care of this @captain-pool

**Comment by captain-pool** [comment]
@cyfra Please take a look. I've Added the Test and Fake Data for Nsynth and Also Fixed This Issue Nsynth is having for Python 3.

**Comment by cyfra** [comment]
Fixed in #184
